docs/documentation/Realtime/recent additions.md

```diff
@@ -35,10 +20,36 @@ Join the discussions on <a href="https://github.com/google/transit/pulls" target
 </div>
 <div class="row"></div> -->
 
-## Recently Merged Proposals &ensp;<img src="../../assets/pr-merged.svg" style="height:1em;"/>
+## Recently Merged Proposals &ensp;<img src="../../../assets/pr-merged.svg" style="height:1em;"/>
```

Comment:
Nit: Maybe "recently _**adopted**_ proposals?"

_Merge_ is a bit GitHub jargon.